api_handlers.go

```diff
@@ -57,7 +58,15 @@ func apiTestRunsHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	var testRuns []TestRun
-	baseQuery := datastore.NewQuery("TestRun").Order("-CreatedAt").Limit(1)
+	var limit int
+	if limit, err = ParseMaxCountParam(r); err != nil {
+		http.Error(w, "Invalid 'max-count' param: "+err.Error(), http.StatusBadRequest)
```

Comment:
Aside: I don't know Go, but am wondering if runtime error exist, and if so how easy/hard it is to write error handling code that's untested and will actually blow up in a different way than intended if run. (Reminds me of a C++ code base I worked on with a lot of out-of-memory handling that was untested.)

Reply:
Some reading for your leisure time: https://blog.golang.org/defer-panic-and-recover